### ISSUE TITLE
[1.19] Crash when trying to open on a server

### ISSUE THREAD

**Issue by Illios42** [EdwinMindcraft/origins-architectury#207]
Hey! First of all, I'd like to say thanks for maintaining this port, me and my staff team are very grateful :D

Now our problem:
We wanted to move to 1.19 soon and tried to load our server with origins-forge-1.19-1.5.0.2
and caelus-forge-1.19-3.0.0.3 (Pehkui 3.3.3 too if that matters, and some other mods I could provide a list of), but it crashes. Here's the crash log:
https://pastebin.com/Z5phQ6UE
 
 I know it's still a beta so ofc issues were to be expected, hope you can fix them!

**Comment by EdwinMindcraft** [comment]
Right, here's the deal: The code works in dev (SP and MP) and in SP forge, so it basically means that, for all intents and purposes, this is either a mod interaction or a problem with JarJar.
For the former, please try to reproduce the bug with only origins and caelus installed.
If you can reproduce the bug, try with the files not ending with `-fatjar.jar` in the [github release](https://github.com/EdwinMindcraft/origins-architectury/releases/latest), you'll need all four files (additionalentityattributes, calio, apoli and origins).

* If you can reproduce the bug with this configuration, please attach the crash report (if available, or latest/debug.log if not).
* If you can't reproduce the bug in this configuration, please state so, as it will most likely be a bug with a forge library.

**Comment by Illios42** [comment]
Thanks for your swift response!
Only Origins and Caelus resulted in an endless loading screen in the fresh world I started. Here's the latest.log https://pastebin.com/0rMEegvC

Tried the other option with the files without `-fatjar.jar` and the world loaded properly and the origins seem to function alright.

**Comment by EdwinMindcraft** [comment]
The endless loading screen is a bug from 1.5.0.2, which should be fixed now in 1.5.0.4 (which is the release you tested by downloading each mod separately).
This is almost certainly a mod interaction, as such, I'd be grateful if you could provide a mod list or try to find exactly which mods are incompatible.
As far as I can tell, this is a crash relating to a mod that alters either fluid movement, digging or fog / general world drawing stuff, if that helps.

**Comment by Illios42** [comment]
I should have mentioned that we tested the new release on our server afterwards (I don't have direct access so I am kinda the middle man here) and it also loaded fine so perhaps the crash was related to the endless loading screen bug? 

Nonetheless, here's the full mod list from when the crash first occurred. The new release works with these as well.

- AdditionalBanners-Forge-1.19-8.0.2
- Adorn-3.6.11.19-forge
- alexsmobs-1.19.3
- AutoRegLib-1.8-54
- backpacked-2.1.9-1.19
- BiomesOPlenty-1.19-17.0.0.143
- Bookshelf-Forge-1.19-14.1.7
- citadel-1.12.6-1.19
- curios-forge-1.19-5.1.0.4
- DramaticDoors-Forge-1.19-2.1.1
- ecologics-forge-1.19-2.1.4
- ellemes-container-library-2.0.41.19-forge
- EnchantingInfuser-v4.0.1-1.19-Forge
- expandedstorage-8.0.21.19-forge
- furnish-1.19-0.6
- kotlinforforge-3.6.0-obf
- lootr-1.19-0.3.20.57
- macawsbridgesbop-1.19-1.0
- mcw-bridges-2.0.3-mc1.19
- mcw-doors-1.0.6-mc1.19
- mcw-fences-1.0.5-mc1.19
- mcw-lights-1.0.3-mc1.19
- mcwfencesbop-1.19-1.1
- oldguns-1.19-3.6.1-33
- Patchouli-1.19-75
- Pehkui-3.3.31.19-forge
- portablejukebox-forge-1.19-2.2.0
- PuzzlesLib-v4.0.17-1.19-Forge
- Quark-3.3-test-367 (Which is not an official release, but an unofficial, modified Beta due to an unrelated crash and the dev being on break)
- repurposed_structures_forge-6.1.41.19
- TerraBlender-forge-1.19-2.0.0.120
- TradingPost-v4.0.0-1.19-Forge
